soc/arm/rpi_pico/rp2/Kconfig.soc

```diff
@@ -0,0 +1,28 @@
+# Raspberry Pi RP2XXX MCU line
+
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2021 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "RP2xxx MCU Selection"
+	depends on SOC_SERIES_RP2XXX
+
+config SOC_RP2040
+	bool "Raspberry Pi RP2040"
+
+endchoice
+
+# Flash type used by the SoC. The board should select the one used.
```

Comment:
Originally I also did it with a choice, but I was requested not to (and I was convinced that it was wrong) because choice requires a prompt. The flash type is not user configurable - it's selected by the board.

Reply:
I had thought that if the board provided a default the prompt became optional, but if not so then fair enough. I though choice was better because it would be bad to end up selecting more than one of them at once.